packages/@aws-cdk/tmp-toolkit-helpers/src/api/diff/diff-formatter.ts

```diff
@@ -184,11 +192,12 @@ export class DiffFormatter {
 
   private formatStackDiffHelper(
     oldTemplate: any,
-    stackName: string | undefined,
+    stackName: string,
```

Comment:
What if it's an empty string?

Reply:
it shouldn't be. but even if it is, i don't think my code would fail. obviously right now if you somehow call `formatStackDiff()` or `formatSecurityDiff` on the same stack in the same `DiffFormatter`, we'd write over the previous diff object. 

1) we are not calling `formatXxx` multiple times per CLI / toolkit call, so I don't expect this to happen in practice at all.
2) even if we did overwrite, we'd be writing the same object as the `fullDiff` is the same.